installer/installer.py

```diff
@@ -0,0 +1,600 @@
+            if answer[0] != "v":
+                answer = f"v{answer}"
+            _t = [a for a in all_releases if a.version == answer]
+            if len(_t) > 0:
+                self.release = _t[0]
+            else:
+                failure_exit(f"Version: {answer} is not exist")
+
+    @staticmethod
+    def default_answer(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwds):
+            _default = kwds.get('default', "")
+            if _default:
+                args = list(args)
+                args[-1] += f"[{_default}] {os.linesep}"
+            value = func(*args)
+            return value if value != "" else _default
+
+        return wrapper
+
+    @default_answer
+    def ask(self, question, **kwargs):
+        return str(input(question))
+
+    def ask_for_home_directory(self, default) -> str:
+        answer = self.ask(
+            f"Please, type here the path to home directory for user cheqd. For keeping default value, just type "
+            f"'Enter': ", default=default)
+        self.home_dir = answer
+
+    def ask_for_cosmovisor(self, default) -> str:
+        answer = self.ask(
+            f"Do you want to use Cosmovisor? "
+            f"Please type any kind of variants: yes, no, y, n. ", default=default)
+        self.is_cosmo_needed = True if answer.lower() in ['yes', 'y'] else False
+
+    def ask_for_init_from_snapshot(self):
+        answer = self.ask(
+            f"Do you want to deploy the latest snapshot? "
+            f"Please type any kind of variants: yes, no, y, n. ",
+            default="No"
+        )
+        self.init_from_snapshot = True if answer.lower() in ['yes', 'y'] else False
+
+    def ask_for_chain(self):
+        answer = self.ask(
+            f"Which chain do you want to use? Possible variants are: {', '.join(DEFAULT_CHAINS)} ",
+            default="testnet"
+        )
+        self.chain = answer if answer in DEFAULT_CHAINS else failure_exit(f"Possible chains are: {DEFAULT_CHAINS}")
+
+    def ask_for_snapshot_url(self):
+        answer = self.ask(
+            f"Which snapshot do you want to use? Please type the full URL to archive or press return to use the latest ",
+            default=self.prepare_url_for_latest()
+        )
+        self.snapshot_url = answer
+
+    def ask_for_setup(self):
+        answer = self.ask(
+            f"Do you want to setup node after installation? "
+            f"Please type any kind of variants: yes, no, y, n. ",
+            default="No"
+        )
+        self.is_setup_needed = True if answer.lower() in ['yes', 'y'] else False
+
+    def ask_for_moniker(self):
+        answer = self.ask(
+            f"Please, type the moniker for your node: {os.linesep}",
+            default=""
+        )
+        self.moniker = answer
+
+    def ask_for_external_address(self):
+        answer = self.ask(
+            f"What is external IP address for your node? Please type in format: <ip_address>:<port>{os.linesep}",
```

Comment:
This seems a bit ambiguous. Maybe we should add a short description that this is a P2P port, like in the readme?